backend/src/main.cpp

```diff
@@ -19,17 +20,32 @@ int main()
     db.execute("INSERT INTO passwords (password) VALUES ('chocolate1');");
     db.printTable("passwords");
 
-    // declare crow application
-    crow::SimpleApp app;
+    // Enable CORS
+    crow::App<crow::CORSHandler> app;
+
+    // Customize CORS
+    auto &cors = app.get_middleware<crow::CORSHandler>();
+
+    // clang-format off
+    cors
+      .global()
+        .headers("*")
+        .methods("POST"_method, "GET"_method)
+      .prefix("/nocors")
+        .ignore();
+    // clang-format on
```

Comment:
**question**: Why do you disable clang formatting?

Reply:
It was from following the (Crow-Cors example online)[https://github.com/CrowCpp/Crow/blob/master/examples/middlewares/example_cors.cpp]. I think by disabling clang formatting it makes it easier to see the different prefixes (global() vs prefix("/nocors")). But honestly I am going to delete .prefix("/nocors").ignore() since I took it out of the endpoints anyways. Since we will only have one endpoint I will delete the comments that take away the clang formatting.